terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,15 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+log_level = "debug"
+restricted_word_web_version = "1.0"
```

Comment:
Wondering about the `restricted_word_web_version` and `restricted_word_web_port` settings. Can't see those set in Mesos. Why are they here?

Reply:
Removed